src/ess/sans/normalization.py

```diff
@@ -341,13 +291,50 @@ def _normalize(
     denominator: sc.DataArray,
     return_events: ReturnEvents,
     uncertainties: UncertaintyBroadcastMode,
-    wavelength_bands: ProcessedWavelengthBands,
 ) -> sc.DataArray:
     """
-    Perform normalization of counts as a function of Q.
-    If the numerator contains events, we use the sc.lookup function to perform the
+    Perform normalization of counts as a function of Q to obtain I(Q) or I(Qx, Qy).
     division.
 
+    In a SANS experiment, the scattering cross section :math:`I(Q)` is defined as
+    (`Heenan et al. 1997 <https://doi.org/10.1107/S0021889897002173>`_):
+
+    .. math::
+
+       I(Q) = \\frac{\\partial\\Sigma{Q}}{\\partial\\Omega} = \\frac{A_{H} \\Sigma_{R,\\lambda\\subset Q} C(R, \\lambda)}{A_{M} t \\Sigma_{R,\\lambda\\subset Q}M(\\lambda)T(\\lambda)D(\\lambda)\\Omega(R)}
+
+    where :math:`A_{H}` is the area of a mask (which avoids saturating the detector)
+    placed between the monitor of area :math:`A_{M}` and the main detector.
+    :math:`\\Omega` is the detector solid angle, and :math:`C` is the count rate on the
+    main detector, which depends on the position :math:`R` and the wavelength.
+    :math:`t` is the sample thickness, :math:`M` represents the incident monitor count
+    rate for the sample run, and :math:`T` is known as the transmission fraction.
+
+    Note that the incident monitor used to compute the transmission fraction is not
+    necessarily the same as :math:`M`, as the transmission fraction is usually computed
+    from a separate 'transmission' run (in the 'sample' run, the transmission monitor is
+    commonly moved out of the beam path, to avoid polluting the sample detector signal).
+
+    Finally, :math:`D` is the 'direct beam function', and is defined as
+
+    .. math::
+
+       D(\\lambda) = \\frac{\\eta(\\lambda)}{\\eta_{M}(\\lambda)} \\frac{A_{H}}{A_{M}}
+
+    where :math:`\\eta` and :math:`\\eta_{M}` are the detector and monitor
+    efficiencies, respectively.
+
+    Hence, in order to normalize the main detector counts :math:`C`, we need compute the
+    transmission fraction :math:`T(\\lambda)`, the direct beam function
+    :math:`D(\\lambda)` and the solid angle :math:`\\Omega(R)`.
+
+    The denominator is then simply:
+    :math:`M_{\\lambda} T_{\\lambda} D_{\\lambda} \\Omega_{R}`,
+    which is equivalent to ``wavelength_term * solid_angle``.
+    The ``wavelength_term`` includes all but the ``solid_angle`` and is computed by
+    :py:func:`iofq_norm_wavelength_term_sample` or
+    :py:func:`iofq_norm_wavelength_term_background`.
```

Comment:
```suggestion
    :math:`M_{\\lambda} T_{\\lambda} D_{\\lambda} \\Omega_{R}`.
```

Maybe removing all references to other code in the docstring is best?